remotespark/datawidgets/autovizwidget.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) 2015  aggftw@gmail.com
+# Distributed under the terms of the Modified BSD License.
+
+import pandas as pd
+from ipywidgets import FlexBox
+from IPython.display import display
+
+from .encoding import Encoding
+from .encodingwidget import EncodingWidget
+from .ipywidgetfactory import IpyWidgetFactory
+
+
+class IpythonDisplay(object):
+    @staticmethod
+    def display_to_ipython(to_display):
+        display(to_display)
+
+
+class AutoVizWidgetTest(FlexBox):
```

Comment:
I don't know if splitting this into two classes is the cleanest way to accomplish this.  It looks like an AutoVizWidget is the same thing as an AutoVizWidgetTest, except with fewer arguments in the constructor? Why not just use more default arguments for those purposes?